Improve Failure to Proof and Production pages (#624)
18F/identity-dev-docs: _pages/user-experience/failure-proof.md

---

## Identity Verification: Create a Failure to Proof URL

> Applications using Login.gov's identity verification are required to provide an alternative method of accessing services to support users who are unable to complete Login.gov’s identity verification process. This is based on existing [OMB M-23-22 policy](https://bidenwhitehouse.archives.gov/wp-content/uploads/2023/09/M-23-22-Delivering-a-Digital-First-Public-Experience.pdf) that requires agencies to offer these alternative channels for access to services.

Reviewer: ```suggestion
Applications using Login.gov's identity verification services are required to provide an alternative method of accessing agency services to support users who are unable to complete Login.gov’s identity verification process. This is based on existing [OMB M-23-22 policy](https://bidenwhitehouse.archives.gov/wp-content/uploads/2023/09/M-23-22-Delivering-a-Digital-First-Public-Experience.pdf) that requires agencies to offer these alternative channels for access to services.
```

> Our Alternative IdV Playbook, which is provided to you during onboarding, explains this in detail. This page serves as a high-level overview.

Reviewer: ```suggestion
Our Alternative IdV Playbook, which we provided to you during onboarding, explains this in detail. This page serves as a high-level overview.
```

Author: I originally had "we", but changed it because this could be read by people who haven't been onboarded yet, so they will be aware that this playbook will be shared with them once they are onboarded.

Reviewer: Makes sense. Thanks for clarifying!

### What should the Failure to Proof page contain?

Below are the main points the Failure to Proof page should address:

- Explain that the user ended up on that page because they were not able to verify their identity with Login.gov

- Link to Login.gov's [identity verification help articles](https://www.login.gov/help/verify-your-identity/overview/) and [Contact](https://www.login.gov/contact/) page to see if Login.gov is able to help the user with their verification issues.

- If there’s nothing more Login.gov can do, provide a way to contact your agency to get help accessing services, and/or a link to the alternative method for accessing services

You can find examples of Failure to Proof pages that meet our guidelines in our Alternative IdV Playbook.

### How does a user end up on the Failure to Proof page?

A user will only end up on your Failure to Proof page if they manually choose to exit out of the Login.gov identity verification process. Throughout the process, there is a "Cancel" link the user can click, which will take them to a screen with three options:

1) Continue the verification process
2) Start over from the beginning
3) Exit Login.gov

> If they choose to exit Login.gov, we will send them to the Failure to Proof URL that you configured in the Partner Portal.

Reviewer: ```suggestion
If the user chooses to exit Login.gov, we will send them to the Failure to Proof URL that you configured in the Partner Portal.
```
